<launch>
	<rosparam file="$(find cepheus_control)/config/cepheus_control.yaml" command="load"/>
	<node name="controller_spawner" pkg="controller_manager"
		type="spawner" respawn="false"
		output="screen" ns="/cepheus"
		args="joint_state_controller left_shoulder_position_controller
		left_elbow_position_controller reaction_wheel_velocity_controller p1_position_controller p2_position_controller"/>
	<node name="robot_state_publisher" pkg="robot_state_publisher"
		type="robot_state_publisher" respawn="false" output="screen">
		<param name="robot_description" command="$(find xacro)/xacro '$(find
			cepheus_description)/urdf/cepheus.xacro'" />
		<remap from="/joint_states" to="/cepheus/joint_states" />
	</node>
	<node name="cepheus_controller" pkg="cepheus_control" type="cepheus_controller_node" output="screen" required="true">
	</node>
</launch>
